assets/cluster-bootstrap/apiserver-apirequestcounts-crd.yaml

```diff
@@ -0,0 +1,254 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml is the version used in OpenShift 4.8.  Please use this version.

Reply:
I was used the crd provided by Cesar from Our Chat. The one he had given was from Master branch. I have updated back the CRD to use 
https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml